api/client.gen.go

```diff
@@ -40,11 +40,27 @@ const (
 
 // Defines values for JobStatus.
 const (
-	JobStatusCANCELLED JobStatus = "CANCELLED"
-	JobStatusFAILED    JobStatus = "FAILED"
-	JobStatusRUNNING   JobStatus = "RUNNING"
-	JobStatusSUBMITTED JobStatus = "SUBMITTED"
-	JobStatusSUCCEEDED JobStatus = "SUCCEEDED"
+	JobStatusCANCELLED         JobStatus = "CANCELLED"
```

Comment:
nice. let's make a note to regenerate this from the landed rerun.yml when that PR on the rerun side lands